src/server/worldserver/worldserver.conf.dist

```diff
@@ -1303,12 +1303,24 @@ Instance.ResetTimeHour = 4
 Instance.UnloadDelay = 1800000
 
 #
-#    Quests.EnableQuestTracker
+#    QuestTracker.Enable
 #        Description: Store data in the database about quest completion and abandonment to help finding bugged quests.
 #        Default:     0  - (Disabled)
 #                     1  - (Enabled)
+#
+#    QuestTracker.Queue.Enable
+#        Description: Enable queue
```

Comment:
what does `Enable queue` mean? @Winfidonarleyan can you please add a more detailed description?

Reply:
Need help for this desc :D